app/controllers/orders_controller.rb

```diff
@@ -21,35 +22,57 @@ def update
     render :show
   end
 
+  def index
+    @merchant = User.find(params[:user_id])
 
-  def destroy
-    @order.destroy
-  end
+    @all_items = @merchant.order_items
 
-  def index
-    @all_orders = Order.all
+    @pend = @all_items.where(status: "pending")
+    @paid = @all_items.where(status: "paid")
+    @ship = @all_items.where(status: "complete")
+
+    @pend_total = @all_items.where(status: "pending").pluck(:total_price)
```

Comment:
You are reusing the logic from lines 30-32 in these totals, can you just do:
`@pend.pluck(:total_price)` 
or something along those lines instead of duplicating the logic?